docs/extend/authorization.md

```diff
@@ -0,0 +1,120 @@
+<!--[metadata]>
++++
+title = "Access authorization plugin"
+description = "How to create authorization plugins to manage access control to your Docker daemon."
+keywords = ["security, authorization, authentication, docker, documentation, plugin, extend"]
+[menu.main]
+parent = "mn_extend"
+weight = -1
++++
+<![end-metadata]-->
+
+
+# Access authorization
+
+By default Docker’s authorization model is all or nothing. Anyone who has access to the Docker daemon has the ability to run any Docker command. Authorization plugins enable creating granular access policies for managing access to Docker daemon. 
+
+## Basic principles
+
+* The authorization sub-system is based on Docker's [Plugin API](http://docs.docker.com/engine/extend/plugin_api), so that anyone could add / develop an authorization plug-in. It is possible to add a plug-in to a deployed Docker daemon without the need to rebuild the daemon. Daemon restart is required.
+
+* The authorization sub-system enables approving or denying requests to the Docker daemon based on the current user and command context, where the authentication context contains all user details and the authentication method, and the command context contains all the relevant request data.
```

Comment:
Yes, agree

Dima Stopel | e: dima at twistlock dot com | p: +972.545.959.320 |
gpg: 71FF 77BA 8989 2CCD



On Mon, Dec 7, 2015 at 3:27 PM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In docs/extend/authorization.md
> <https://github.com/docker/docker/pull/15365#discussion_r46819718>:
>
> > +[menu.main]
> > +parent = "mn_extend"
> > +weight = -1
> > ++++
> > +<![end-metadata]-->
> > +
> > +
> > +# Access authorization
> > +
> > +By default Docker’s authorization model is all or nothing. Anyone who has access to the Docker daemon has the ability to run any Docker command. Authorization plugins enable creating granular access policies for managing access to Docker daemon.
> > +
> > +## Basic principles
> > +
> > +* The authorization sub-system is based on Docker's [Plugin API](http://docs.docker.com/engine/extend/plugin_api), so that anyone could add / develop an authorization plug-in. It is possible to add a plug-in to a deployed Docker daemon without the need to rebuild the daemon. Daemon restart is required.
> > +
> > +* The authorization sub-system enables approving or denying requests to the Docker daemon based on the current user and command context, where the authentication context contains all user details and the authentication method, and the command context contains all the relevant request data.
>
> probably better to use @moxiegirl <https://github.com/moxiegirl>'s
> example; I think she did a whole rewrite of the docs that you can copy/paste
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/15365/files#r46819718>.
>